app/scripts/lib/buy-url.js

```diff
@@ -119,17 +119,20 @@ const createMoonPayUrl = async (walletAddress, chainId) => {
  *
  * @param {string} walletAddress - Ethereum destination address
  * @param {string} chainId - Current chain ID
+ * @param {string|undefined} symbol - Token symbol to buy
  * @returns String
  */
-const createCoinbasePayUrl = (walletAddress, chainId) => {
-  const { coinbasePayCurrencies } = BUYABLE_CHAINS_MAP[chainId];
+const createCoinbasePayUrl = (walletAddress, chainId, symbol) => {
+  // since coinbasePayCurrencies is going to be extended to include all tokens supported
+  // we now default to nativeCurrency instead of the 2 previous tokens + eth that we had before
+  const { nativeCurrency } = BUYABLE_CHAINS_MAP[chainId];
   const queryParams = new URLSearchParams({
     appId: COINBASEPAY_API_KEY,
     attribution: 'extension',
     destinationWallets: JSON.stringify([
       {
         address: walletAddress,
-        assets: coinbasePayCurrencies,
+        assets: symbol ? [symbol] : [nativeCurrency],
```

Comment:
I am just concerned if this change will limit the user to select a single asset at the provider checkout page instead of a list of allowed assets, we may need to differentiate between the list of assets and the default selected asset (in our case dictated by the symbol param)

Reply:
Its a frictionless way to buy the selected asset coming from metamask. Theres not a way where we can display both the full list and a selected asset as a default. We either show them a list on the coinbase pay side, or we pre select a particular asset

_With a list it would be this way:_

TokenA -> List of currencies -> Buy Token A

_With this change:_

TokenA -> Buy Token A

Will ask this over slack so everyone can chime in